lua/tardis/settings/sh_global_settings.lua

```diff
@@ -44,6 +44,93 @@ TARDIS:AddSetting({
     name="MaxHealth",
 })
 
+TARDIS:AddSetting({
+    id="long_repair",
+    type="bool",
+    value=true,
+
+    class="global",
+    convar = {
+        name = "tardis2_long_repair",
+        flags = CONVAR_FLAGS,
+        desc = "TARDIS - Whether repairing process lasts 60 seconds",
+    },
+
+    option = true,
+    section=SETTING_SECTION,
+    name="WaitLongForRepair",
+})
+
+TARDIS:AddSetting({
+    id="artron_energy",
+    type="bool",
+    value=true,
+
+    class="global",
+    convar = {
+        name = "tardis2_artron_energy",
+        flags = CONVAR_FLAGS,
+        desc = "TARDIS - Whether artron energy is enabled",
+    },
+
+    option = true,
+    section=SETTING_SECTION,
+    subsection="ArtronEnergy",
+    name="ArtronEnergyEnabled",
+})
+
+TARDIS:AddSetting({
+    id="artron_energy_start",
+    type="list",
+    value=0.25,
+    get_values_func = function()
+        return {
+            { "0%", 0.0 },
+            { "25%", 0.25 },
+            { "50%", 0.5 },
+            { "75%", 0.75 },
+            { "100%", 1.0 },
+        }
+    end,
+
+    class="global",
+    convar = {
+        name = "tardis2_artron_energy_start",
+        flags = CONVAR_FLAGS,
+        desc = "TARDIS - Initial amount of artron energy",
+    },
+
+    option=true,
+    section=SETTING_SECTION,
+    subsection="ArtronEnergy",
+    name="ArtronEnergyStart",
+})
+
+TARDIS:AddSetting({
+    id="artron_energy_max",
+    type="integer",
+    value=14400,
+    min=1800,
+    max=57600,
+    round_func = function(x)
+        if x > 20000 then return (x - x % 7200) end
+        if x > 10000 then return (x - x % 3600) end
+        return (x - x % 1800)
+    end,
+
+    class="global",
+    convar = {
+        name = "tardis2_artron_energy_max",
+        flags = CONVAR_FLAGS,
+        desc = "TARDIS - Max amount of artron energy",
+    },
+
+    option = true,
+    section=SETTING_SECTION,
+    subsection="ArtronEnergy",
+    name="ArtronEnergyMax",
+})
+
```

Comment:
Should we have an option for charging speed multipler? I think combined with these other two options that will give players/admins enough control over energy to do what they like, max health already effectively controls the decrease rate.

Reply:
I don't really like this idea
The point of artron was making people use handbrake and add some difficulty to the game
(allowing them to disable it if they want)

I think with such an option everyone will just make it recharge quickly and it will lose its whole point

Reply:
I'll discuss this with you further on discord I think it is a good idea still

Reply:
I've been giving it some thought and decided to make it in a different form
See 30340f729ea6f23cac310a1f3a44914406d5d014 and 8db80d0c985561170913e601499f62fd038f8218

It regulates the charging speed but doesn't let the user abuse it too much, allowing to select between 2 and 10 seconds for normal non-handbrake charging time (5 is default)

Handbrake charges it every second and does it rather quickly, I don't think we should modify the speed with it